airflow/providers/google/cloud/operators/dataproc_metastore.py

```diff
@@ -29,13 +29,121 @@
 from googleapiclient.errors import HttpError
 
 from airflow import AirflowException
-from airflow.models import BaseOperator
+from airflow.models import BaseOperator, BaseOperatorLink
+from airflow.models.taskinstance import TaskInstance
 from airflow.providers.google.cloud.hooks.dataproc_metastore import DataprocMetastoreHook
+from airflow.providers.google.common.links.storage import StorageLink
 
 if TYPE_CHECKING:
     from airflow.utils.context import Context
 
 
+BASE_LINK = "https://console.cloud.google.com"
+METASTORE_BASE_LINK = BASE_LINK + "/dataproc/metastore/services/{region}/{service_id}"
+METASTORE_BACKUP_LINK = METASTORE_BASE_LINK + "/backups/{backup_id}?project={project_id}"
+METASTORE_BACKUPS_LINK = METASTORE_BASE_LINK + "/backuprestore?project={project_id}"
+METASTORE_EXPORT_LINK = METASTORE_BASE_LINK + "/importexport?project={project_id}"
+METASTORE_IMPORT_LINK = METASTORE_BASE_LINK + "/imports/{import_id}?project={project_id}"
+METASTORE_SERVICE_LINK = METASTORE_BASE_LINK + "/config?project={project_id}"
+
+
+class DataprocMetastoreBackupLink(BaseOperatorLink):
+    """Helper class for constructing Dataproc Metastore Backup link"""
+
+    name = "Dataproc Metastore Backup"
+
+    def get_link(self, operator, dttm):
+        ti = TaskInstance(task=operator, execution_date=dttm)
+        backup_conf = ti.xcom_pull(task_ids=operator.task_id, key="backup_conf")
```

Comment:
Let's should use `XCom.get_one()` instead of creating ad hoc `TaskInstance` objects to pull the necessary `XCom` for operator links. Check out #21285. Using the `XCom.get_one()` interface is also more straightforward and pertinent in these case.

Reply:
@josh-fell To adjust changes for the Metastore links we need to wait when your PR will be merged 😄  you changed the types in the `get_one` method also 😄 without this `pre-commit` will fail

Reply:
@lwyszomi  - merged :). You can rebase now.

Reply:
@potiuk Thank you, we will update also other PRs where we added new links :)